tests/integration/_validator/testcases/data/validator.yaml

```diff
@@ -17,7 +15,7 @@ airgapConfig:
   enabled: false
 kindConfig:
   useKindCluster: true
-  kindClusterName: "<kind_cluster_name>"
+  kindClusterName: "validator-kind-cluster-60e58"
```

Comment:
did you mean to check this in? guess it doesn't really matter since the cluster name will just be auto generated

Reply:
Yeah - I did notice it in the diff but also felt it didn't matter and was too lazy to revert it..